app/controllers/users/dashboard_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Users
+  class DashboardController < ApplicationController
+    def index
+      @current_day_of_week = Date.today.on_weekday? ? Date.today.strftime('%A') : 'Monday'
```

Comment:
'Monday' should totally be wrapped in a constant `default_day`. But do that in a facade